### PR TITLE
ci: Add cache for qemu experimental

### LINKED ISSUE #1967: Cache for qemu and kernel for virtiofs 3.0

### PR DESCRIPTION
**Pull request by GabyCT**
This will build qemu experimental using a cache jenkins job.

Fixes #1967

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: .ci/install_qemu_experimental.sh [hunk 1]
```diff
@@ -25,16 +25,27 @@ arch=$("${cidir}"/kata-arch.sh -d)
...
+	sha256sum -c "sha256sum-${QEMU_TAR}" || return 1
+	uncompress_static_qemu "${QEMU_TAR}"
+	sudo -E ln -s "${QEMU_PATH}" "/usr/bin"
+	sudo -E ln -s "${VIRTIOFS_PATH}" "/usr/bin"
+}
+
 build_and_install_static_experimental_qemu() {
 	build_experimental_qemu
-	uncompress_experimental_qemu "${QEMU_TAR}"
+	uncompress_experimental_qemu "${KATA_TESTS_CACHEDIR}"/"${QEMU_TAR}"
```
Comment (chavafg) [inline .ci/install_qemu_experimental.sh]: this could be: `"${KATA_TESTS_CACHEDIR}/${QEMU_TAR}"`

File: .ci/ci_cache_components.sh [hunk 2]
```diff
@@ -146,6 +155,7 @@ Usage: $script_name [options]
 Description: This script builds the cache of several Kata components.
 Options:
 
+-a	Run qemu experimental cache
```
Comment (chavafg) [inline .ci/ci_cache_components.sh]: indentation not aligned with below lines

File: .ci/ci_cache_components.sh [hunk 3]
```diff
@@ -158,8 +168,11 @@ EOT
 
 main() {
 	local OPTIND
-	while getopts "hiknqr" opt; do
+	while getopts "ahiknqr" opt; do
 		case "$opt" in
+	       a)
```
Comment (chavafg) [inline .ci/ci_cache_components.sh]: indentation issue

**Comment by GabyCT** [comment]
@chavafg changes applied

**Pull request merged**